cmd/wio/commands/run/dependencies/flags.go

```diff
@@ -60,28 +60,40 @@ func fillGlobal(givenFlags, requiredFlags []string) ([]string, error) {
                 goto Continue
             }
         }
-        return nil, errors.Stringf("global flag %s unfilled", required)
+        return nil, errors.Stringf("global flag/definition %s unfilled in %s", required, name)
 
     Continue:
         continue
     }
+
+    if len(givenFlags) < len(requiredFlags) {
```

Comment:
Do we need this case? If there are less given flags than required flags, then at least one required flag will not be filled in the loop above, so this statement will never be true

Reply:
The is needed when no flag is given. But I added this statement to handle any other case as well. I had some tests break cuz of this